opa-manager-operator/files/pod-spec.yaml.jinja2

```diff
@@ -0,0 +1,319 @@
+          resources:
+            - '*'
+          verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+        - apiGroups:
+            - templates.gatekeeper.sh
+          resources:
+            - constrainttemplates
+          verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+        - apiGroups:
+            - templates.gatekeeper.sh
+          resources:
+            - constrainttemplates/finalizers
+          verbs:
+            - delete
+            - get
+            - patch
+            - update
+        - apiGroups:
+            - templates.gatekeeper.sh
+          resources:
+            - constrainttemplates/status
+          verbs:
+            - get
+            - patch
+            - update
+        - apiGroups:
+            - admissionregistration.k8s.io
+          resourceNames:
+            - gatekeeper-validating-webhook-configuration
+          resources:
+            - validatingwebhookconfigurations
+          verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+
+containers:
+  # - envConfig:
+  #     POD_NAMESPACE:
+  #       field:
+  #         path: metadata.namespace
+  #         api-version: v1
+  #     POD_NAME:
+  #       field:
+  #         path: metadata.name
+  #         api-version: v1
+  #   imageDetails:
+  #     imagePath: {{ image_path }}
+  #   ports:
+  #     - containerPort: 8889
+  #       name: audit-metrics
+  #       protocol: TCP
+  #     - containerPort: 9091
+  #       name: audit-healthz
+  #       protocol: TCP
+  #   imagePullPolicy: {{ imagePullPolicy }}
+  #   name: {{ app_name }}-audit
+  #   args: {{ audit_cli_args }}
+  #   command:
+  #     - /manager
+  #   image: 'openpolicyagent/gatekeeper:v3.2.3'
+  #   kubernetes:
+  #     livenessProbe:
+  #       httpGet:
+  #         path: /healthz
+  #         port: 9091
+  #     readinessProbe:
+  #       httpGet:
+  #         path: /readyz
+  #         port: 9091
+  #     securityContext:
+  #       allowPrivilegeEscalation: false
+  #       capabilities:
+  #         drop:
+  #           - all
+  #       runAsGroup: 999
+  #       runAsNonRoot: true
+  #       runAsUser: 1000
+  - envConfig:
+      POD_NAMESPACE:
+        field:
+          path: metadata.namespace
+          api-version: v1
+      POD_NAME:
+        field:
+          path: metadata.name
+          api-version: v1
+    imageDetails:
+      imagePath: {{ image_path }}
+    ports:
+    - containerPort: 8443
+      name: webhook-server
+      protocol: TCP
+    - containerPort: 8888
+      name: metrics
+      protocol: TCP
+    - containerPort: 9090
+      name: healthz
+      protocol: TCP
+    imagePullPolicy: {{ imagePullPolicy }}
+    name: {{ app_name }}-manager
```

Comment:
I wouldn't bother including app_name in the container name, since the container will be scoped to a pod that includes the app_name anyway.

Reply:
cool